code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -271,3 +271,85 @@
 	if(L.a_intent == "harm" && L.reagents && !stat)
 		L.reagents.add_reagent("nutriment", 0.4)
 		L.reagents.add_reagent("vitamin", 0.4)
+
+/mob/living/simple_animal/pet/cat/ascendant // Now you silly admins can summon the Catlord's true form. Or turn me into it!

```

Comment:
how much you wanna bet if I google his ckey I find some self insert porn or something